third_party/silabs/efr32_sdk.gni

```diff
@@ -91,6 +91,15 @@ declare_args() {
 
   # Enable TestEventTrigger in GeneralDiagnostics cluster
   sl_enable_test_event_trigger = false
+
+  # Temperature Sensor support
+  sl_enable_si70xx_sensor = false
+}
+
+declare_args() {
+  # This argument is for retro-compatibility since use_temp_sensor was used by the thermostat app.
+  # It should not be used anymore.
+  use_temp_sensor = sl_enable_si70xx_sensor
```

Comment:
why would we need this still?

Reply:
Really left it in case a customer has their product use that flag. I can remove it we don't really care about that use-case.

Reply:
removing the old flags